nemo_rl/data/datasets/response_datasets/squad.py

```
@@ -12,35 +12,47 @@
 class SquadDataset(RawDataset):
-    def __init__(self) -> None:
-        original_ds = load_dataset("rajpurkar/squad")
-        self.task_name = "SQuAD"
-        self.formatted_ds = original_ds.map(format_squad)
+    """Simple wrapper around the squad dataset.
+
+    Args:
+        split: Split name for the dataset, default is "train"
+    """
+
+    def __init__(self, split: str = "train", **kwargs) -> None:
```

Ruff (0.14.13)

Unused method argument: `kwargs`

(ARG002)